core/src/trezor/lvglui/i18n/locales/zh_hk.py

```
@@ -1042,11 +1042,14 @@
     "助記詞的單詞數量並不會影響安全性，所有助記詞在加密層麵上都是安全的。",
     "顯示動態二維碼",
     "BTC, TRON, SOL, ETH, BNB ...",
-    "部分加密資產與硬體功能在二維碼連接模式下無法使用。 \n此模式僅適用於極少數不常操作硬體錢包的使用者，且無法與其他連接方式相容。 \n若您希望透過藍牙或 USB 連接硬體錢包，請重新新增錢包以切換連接模式。",
+    "部分加密資產與硬體功能在二維碼連接模式下無法使用。 \n\n此模式僅適用於極少數不常操作硬體錢包的使用者，且無法與其他連接方式相容。 \n\n若您希望透過藍牙或 USB 連接硬體錢包，請重新新增錢包以切換連接模式。",
```

Ruff (0.14.7)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)